config/api.yml

```diff
@@ -461,6 +461,9 @@
         :identifier: miq_report_run
     :subcollections:
     - :results
+    :collection_actions:
+      :post:
+      - :name: import
```

Comment:
@imtayadeway @abellotti Yes, that feature ID allows both report import and export.